evalme/classification.py

```diff
@@ -30,16 +30,19 @@ def exact_match(self, item, label_weights=None, per_label=False):
                              f'Reason: {exc}', exc_info=True)
                 # different types of results:
                 if per_label:
-                    return {'Error': 0}
+                    return {}
                 else:
                     return 0
             if not isinstance(labels, list):
                 labels = [labels]
-
             if not isinstance(y_labels, list):
                 y_labels = [y_labels]
+            # Check if spans are mismatched
+            mismatched_spans = False
+            if EvalItem.has_spans([x, y]):
```

Comment:
it's not ideal solution - what if choices will be attached to other annotations, boxes, polygons, etc. ?